### ISSUE TITLE
support subproperties in Edges box, based on ontology relationships

### ISSUE THREAD

**Issue by SusanBrown** [cwrc/HuViz#192]
We want the edges, like the classes, to support the nested structures of the ontology

## Expected Behaviour
I would expect that the subproperties of religion shown below would nest inside religion and perhaps be collapsed by default and certainly collapsible into religion, so as to simplify the picklist.

![image](https://user-images.githubusercontent.com/545952/47200405-0af70d80-d344-11e8-9e16-b2f77ca2dd2c.png)

**Comment by SusanBrown** [comment]
This is happening now, is it not @smurp ?

**Comment by smurp** [comment]
Yes, it is happening now but only when the relevant ontology is actually being loaded.  `owl mini` does not support this.  Currently only the loaded ontology has its hierarchy displayed in this way.  

In other words: ideally what we want is fully automatic loading of all ontologies used in the graph (all the way down) and respect for all their hierarchies portrayed in the `pickers`
